.circleci/config.yml

```diff
@@ -42,47 +42,58 @@ jobs:
             docker ps -a
 
       - run:
-          name: Run backend tests
+          name: Run database tests
           when: always
           command: |
-            # Start databases and wait for them to start properly
-            docker-compose -f ./test/docker-compose.yml up -d db-sql-server db-mysql db-mariadb db-postgresql
-            sleep 30
+            # Run database tests
+            docker-compose -f ./test/docker-compose.yml up test-db
 
-            # Run tests
-            docker-compose -f ./test/docker-compose.yml up test
+      - run:
+          name: Run API tests
+          when: always
+          command: |
+            # Run API tests
+            docker-compose -f ./test/docker-compose.yml up test-api
 
       - run:
-          name: Run backend linter
+          name: Run scripts tests
           when: always
           command: |
-            docker run --name mobydq-lint mobydq-test /bin/bash -c "find . -iname \"*.py\" | xargs pylint --output-format=pylint2junit.JunitReporter > lint-results.xml"
+            # Start databases and wait for them to start properly
+            docker-compose -f ./test/docker-compose.yml up -d db-sql-server db-mysql db-mariadb db-postgresql
+            sleep 30
+
+            # Run tests
+            docker-compose -f ./test/docker-compose.yml up test-scripts
 
       - run:
-          name: Collect backend results
+          name: Run scripts linter
           when: always
           command: |
-            docker cp mobydq-test:/srv/test-results.xml ./test-results/
-            docker cp mobydq-lint:/srv/lint-results.xml ./test-results/
+            docker run --name mobydq-test-scripts-lint mobydq-test-scripts /bin/bash -c "find . -iname \"*.py\" | xargs pylint --output-format=pylint2junit.JunitReporter > lint-results.xml"
 
       - run:
-          name: Run frontend tests
+          name: Run app tests
           when: always
           command: |
-            docker run --name mobydq-app-test -e JEST_JUNIT_OUTPUT="/reports/junit/js-test-results.xml" -e CI=true mobydq-app-test npm test -- --ci --reporters=jest-junit --updateSnapshot
+            docker run --name mobydq-test-app -e JEST_JUNIT_OUTPUT="/reports/junit/js-test-results.xml" -e CI=true mobydq-test-app npm test -- --ci --reporters=jest-junit --updateSnapshot
```

Comment:
The docker-compose image name is still mobydq-app-test => Also rename or update this command

Reply:
Do you think the block `app-test` in `/root/docker-compose.test.yml`
Should be moved to `/root/test/docker-compose.yml` so that we have a single compose file to execute tests?

Reply:
Having fewer docker-compose files is handy and I think merging them makes sense, even though I'm not sure which one to keep. I personally like the `docker-compose.test.yml` file